specification/resources/vpcs/vpcs_create_peerings.yml

```
@@ -0,0 +1,63 @@
+operationId: vpcs_create_peerings
+
+summary: Create a Peering with a VPC
+
+description: |
+  To create a new VPC peering for a given VPC, send a POST request to
+  `/v2/vpcs/$VPC_ID/peerings`.
+
+tags:
+  - VPCs
+
+parameters:
+  - $ref: 'parameters.yml#/vpc_id'
+
+requestBody:
+  required: true
+
+  content:
+    application/json:
+      schema:
+        type: object
+        properties:
+          name:
+            type: string
+            pattern: '^[a-zA-Z0-9\-\.]+$'
+            example: nyc1-blr1-peering
+            description: The name of the VPC peering. Must be unique and may only contain
+              alphanumeric characters, dashes, and periods.
+          vpc_id:
+            type: string
+            format: uuid
+            example: c140286f-e6ce-4131-8b7b-df4590ce8d6a
+            description: The ID of the VPC to peer with.
+
+        required:
+        - name
+        - vpc_id
+
+responses:
+  '200':
```

```suggestion
  '202':
```

And the create responses return 202s on success.